internal/ignition/ignition.go

```diff
@@ -342,6 +351,28 @@ func (g *installerGenerator) modifyBMHFile(file *config_31_types.File, bmh *bmh_
 	return nil
 }
 
+func (g *installerGenerator) updateIgnitions() error {
+	masterPath := filepath.Join(g.workDir, "master.ign")
+	caCertFile := g.serviceCACert
+
+	if caCertFile != "" {
+		err := setCACertInIgnition(models.HostRoleMaster, masterPath, g.workDir, caCertFile)
+		if err != nil {
+			return errors.Wrapf(err, "error writing master ignition files")
+		}
+	}
+
+	workerPath := filepath.Join(g.workDir, "worker.ign")
+	if caCertFile != "" {
+		err := setCACertInIgnition(models.HostRoleWorker, workerPath, g.workDir, caCertFile)
+		if err != nil {
+			return errors.Wrapf(err, "error writing worker ignition files")
```

Comment:
Same here, mention that adding the cert is what failed.

Reply:
Done!